tests/errors/cdef_metaclass.pyx

```diff
@@ -0,0 +1,24 @@
+# mode: error
```

Comment:
Just to distinguish it from the other test, it would be good to name this `e_cdef_metaclass.pyx`, "e" for "error".